views/scan.ejs

```diff
@@ -119,13 +119,19 @@
             <td>
               <select name="queryableSelection"<% if (!Table.GlobalSecondaryIndexes) { %> disabled<% } %>>
                 <option value="table"
-                  data-required-query-keys="<%= Table.KeySchema.map(keySchemaItem => keySchemaItem.AttributeName).join(',') %>"
+                  <% Table.KeySchema.forEach(keySchemaItem => { %>
+                    data-key-<%= keySchemaItem.KeyType.toLowerCase() %>="<%= keySchemaItem.AttributeName %>"
+                  <% }) %>
+                  <% if (sortKey) { %> data-sort-key="<%= sortKey.AttributeName %>" <% } %>
```

Comment:
`sortKey is not defined`

Reply:
Sorry about that, that was dead code, deleted that line